pages/qdm-to-qicore.md

```diff
@@ -1424,7 +1453,6 @@ profile address Immunization, Administered.
 | code                         | [MedicationAdministration.medication\[x\]](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.medication[x])       |ֵ Example uses SNOMED substance codes                                     |
 | id                           | [MedicationAdministration.id](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.id)                               |                                                                         |
 | dosage                       | [MedicationAdministration.dosage.dose](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.dose)             | Simple Quantity - Amount of medication for one administration           |
-|                              | [MedicationAdministration.dosage.rate\[x\]](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x])     | Simple Quantity                                                         |
 | route                        | [MedicationAdministration.dosage.route](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.route)           |                                                                         |
 | frequency                    | [MedicationAdministration.request](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.request)                     | Reference to original MedicationRequest with content about prescription |
 |                              | [MedicationAdministration.dosage.rate\[x\]](StructureDefinition-qicore-medicationadministration-definitions.html#medicationadministration.dosage.rate[x])     | Identifies the speed with which the medication was or will be introduced into the patient (e.g., infusion rate).                              |
```

Comment:
I don't think you introduced this broken link, but would you mind resolving it?
Should be: `StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x]`

Reply:
This one, I've seen it labeled as an error, but I don't know why. Is your link not exactly what is already there? The existing link actually works in a browser, too. I don't know what the problem is.

Reply:
Yes, the link does work. However, the case is incorrect. It appears that the validator is case-sensitive. Rightly so, as there are web servers like IIS where urls are case-sensitive.

Current: `StructureDefinition-qicore-medicationadministration-definitions.html#medicationadministration.dosage.rate[x]`

Suggested: `StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x]`

Reply:
Resolved in latest commit